pyatlan/model/assets/process.py

```diff
@@ -289,6 +307,7 @@ def create(
 
 
 from .airflow_task import AirflowTask  # noqa
+from .alteryx_workflow import AlteryxWorkflow  # noqa
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.